### DISCUSSION TITLE
Moch. Nur Faisal | fais

### DISCUSSION THREAD

**Discussion by faisalmoch** [onlyphantom/llm-python#35] [Practicum: Generative AI]
https://colab.research.google.com/drive/1UTZT7QZDOCxMWyZKTtmm_8fdoEGclDFV#scrollTo=i_SdqPeOZkH1

**Comment by onlyphantom** [comment]
Hey! Thank you for the submission! The only nitpick is that your model should still formulate a proper response when the day before isn't a trading day (weekend, public holidays etc) instead of the current response of ` The most traded stocks yesterday were: ...`

The rest of the exercise produce a satisfactory response so I'll mark this as passed, but this is one thing you can improve on! 

```
Question: What are the most traded stock yesterday?


> Entering new AgentExecutor chain...

Invoking: `get_top_companies_by_tx_volume` with `{'start_date': '2024-08-04', 'end_date': '2024-08-04', 'top_n': 3}`


{}The most traded stocks yesterday were:

> Finished chain.
Answer: 
 The most traded stocks yesterday were: 

```
